dev-client/src/constants.ts

```diff
@@ -25,10 +25,10 @@ export const LATITUDE_MAX = 90;
 export const LONGITUDE_MIN = -180;
 export const LONGITUDE_MAX = 180;
 export const SITE_NAME_MIN = 3;
-export const SITE_NAME_MAX = 120;
-export const PROJECT_NAME_MAX_LENGTH = 120;
+export const SITE_NAME_MAX = 50;
+export const PROJECT_NAME_MAX_LENGTH = 50;
 export const PROJECT_NAME_MIN_LENGTH = 3;
-export const PROJECT_DESCRIPTION_MAX_LENGTH = 240;
+export const PROJECT_DESCRIPTION_MAX_LENGTH = 72;
```

Comment:
This seems rather short, I commented on #534 asking Courtney why we want 72, but lmk if you have an opinion

Reply:
Agree this seems too short.

Reply:
Courtney still says 72 ([link](https://github.com/techmatters/terraso-mobile-client/issues/534#issuecomment-2090411784)). I figure we can change it easily later if users want longer.

Reply:
@CourtneyLee333 where did 72 come from? Can we do a round number like 75?